app/genesis.go

```diff
@@ -123,6 +126,22 @@ func NewDefaultGenesisState() GenesisState {
 			WithdrawAddrEnabled: true,
 			PreviousProposer:    nil,
 		},
+		GovData: gov.GenesisState{
+			StartingProposalID: 1,
+			DepositParams: gov.DepositParams{
+				MinDeposit:       sdk.Coins{coin.NewCybCoin(100 * coin.Giga)},
```

Comment:
It must be 1 TCYB

Reply:
it is not final params, I prefer right now to use lower values to test chain.

Reply:
Ok, lets try

Reply:
I did 0.5tcyb, as current top 50 network accs.